### PR TITLE
Add -v option to clean volumes when remove PCA docker containers

### PR DESCRIPTION
**Pull request by mklkun**

### CODE REVIEW

**Review by mboussaa — APPROVED** [review]

**Comment by mboussaa** [comment]
@mklkun other workflows contain already docker rm -fv ? why not doing it everywhere ?

File: JupyterLab/resources/catalog/Finish_JupyterLab.xml [hunk 1]
```diff
@@ -104,6 +104,7 @@ if(engine.toLowerCase().equals("singularity")){
         <![CDATA[ Finish JupyterLab instance and remove its docker container ]]>
       </description>
       <genericInformation>
+        <info name="task.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/jupyterlab.png"/>
```
Comment (fviale) [inline JupyterLab/resources/catalog/Finish_JupyterLab.xml:107]: duplicated line

Comment (mklkun) [reply JupyterLab/resources/catalog/Finish_JupyterLab.xml]: Ok. Did not see that.

**Comment by mklkun** [comment]
> @mklkun other workflows contain already docker rm -fv ? why not doing it everywhere ?

@mboussaa The `Finish_Action.bash` script is shared by all PCA services actually. (Except the Storm and Clearwater scripts that I changed also).

**Review by fviale — APPROVED** [review]

**Pull request merged**